paper/paper.md

```diff
@@ -56,22 +64,22 @@ Here, we propose a software `buildH` to fill this need. `buildH` is very light a
 
 Beyond order parameter calculations, the trajectory with Hs can be be used for any further analyses (e.g. precise molecular volume calculation).
 
-`buildH` has been natively developed for a use in the Unix command line. It possesses a minimum number of options so that it is very easy to use and to learn. It is also possible to use it as a Python module which may be convenient in some cases.
+`buildH` has been natively developed for a use in the Unix command line. It possesses a minimum number of options to be easy to use and learn. It is also possible to use it as a Python module which may be convenient in some cases.
```

Comment:
Sounds weird to me, maybe I'm wrong 😉 I would replace `to be easy to use and learn` by `so that it is easy to use and learn`

Reply:
ahah to me it's the "so that it is.." which sounds weird! 
Another solution: "it possesses a minimum number of options for a simple usage."

Reply:
okay for this one!

Reply:
Excellent 😝